src/main/java/seedu/address/logic/parser/ImportCommandParser.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.logic.commands.ImportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Gender;
+import seedu.address.model.person.MrtStation;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.SecLevel;
+import seedu.address.model.person.Student;
+import seedu.address.model.tag.Subject;
+
+/**
+ * Parses input arguments and creates a new ImportCommand object
+ */
+public class ImportCommandParser implements Parser<ImportCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the ImportCommand
+     * and returns a ImportCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ImportCommand parse(String args) throws ParseException {
+        List<Student> students = new ArrayList<>();
+
+        String projectPath = System.getProperty("user.dir");
+        String path = projectPath + "/" + args.trim();
+
+        File csvFile = new File(path);
+        if (!csvFile.exists()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, ImportCommand.MESSAGE_USAGE));
+        }
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(csvFile));
+            String header = reader.readLine();
+            String[] expectedHeaders = {"Name", "Phone Number", "Email", "Address",
+                "Gender", "Sec level", "Nearest Mrt station", "Subject"};
```

Comment:
I would suggest to set these as constants, as this should match the MESSAGE_USAGE in ImportCommand.java.